test-data.yml

```diff
@@ -0,0 +1,2326 @@
+        index_in: [0, 1, 1, 2]
+        length: 4
+        offsets_in: [0, 2, 3, 3]
+      outputs:
+        mask_out: [0, 1, 1, 3]
+        starts_out: [0, 2, 3, 3]
+        stops_out: [2, 3, 3, 3]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2]
+        length: 4
+        offsets_in: [0, 2, 3, 4]
+      outputs:
+        mask_out: [0, 1, 1, 3]
+        starts_out: [0, 2, 3, 3]
+        stops_out: [2, 3, 3, 4]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2]
+        length: 4
+        offsets_in: [0, 2, 3, 5]
+      outputs:
+        mask_out: [0, 1, 1, 3]
+        starts_out: [0, 2, 3, 3]
+        stops_out: [2, 3, 3, 5]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2]
+        length: 4
+        offsets_in: [0, 2, 3, 5]
+      outputs:
+        mask_out: [0, 1, 2, 3]
+        starts_out: [0, 2, 2, 3]
+        stops_out: [2, 2, 3, 5]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2]
+        length: 4
+        offsets_in: [0, 2, 3, 6]
+      outputs:
+        mask_out: [0, 1, 1, 3]
+        starts_out: [0, 2, 3, 3]
+        stops_out: [2, 3, 3, 6]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2]
+        length: 4
+        offsets_in: [0, 2, 4, 5]
+      outputs:
+        mask_out: [0, 1, 1, 3]
+        starts_out: [0, 2, 4, 4]
+        stops_out: [2, 4, 4, 5]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2]
+        length: 4
+        offsets_in: [0, 3, 3, 4]
+      outputs:
+        mask_out: [0, 1, 2, 3]
+        starts_out: [0, 3, 3, 3]
+        stops_out: [3, 3, 3, 4]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2]
+        length: 4
+        offsets_in: [0, 3, 3, 5]
+      outputs:
+        mask_out: [0, 1, 1, 3]
+        starts_out: [0, 3, 3, 3]
+        stops_out: [3, 3, 3, 5]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2]
+        length: 4
+        offsets_in: [0, 4, 5, 6]
+      outputs:
+        mask_out: [0, 1, 1, 3]
+        starts_out: [0, 4, 5, 5]
+        stops_out: [4, 5, 5, 6]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2]
+        length: 4
+        offsets_in: [0, 4, 5, 6]
+      outputs:
+        mask_out: [0, 1, 2, 3]
+        starts_out: [0, 4, 4, 5]
+        stops_out: [4, 4, 5, 6]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2, 1, 3, 4, 5]
+        length: 8
+        offsets_in: [0, 2, 4, 6, 8, 10, 12]
+      outputs:
+        mask_out: [0, 1, 2, 3, 1, 5, 6, 7]
+        starts_out: [0, 2, 2, 4, 6, 6, 8, 10]
+        stops_out: [2, 2, 4, 6, 6, 8, 10, 12]
+    - error: true
+      inputs:
+        index_in: [0, 1, 1, 2, 3, 4, 5, 6]
+        length: 8
+        offsets_in: [0, 1, 1, 1, 1, 1, 1, 1]
+      outputs:
+        mask_out: [0, 1, 2, 3, 4, 5, 6, 7]
+        starts_out: [0, 1, 1, 1, 1, 1, 1, 1]
+        stops_out: [1, 1, 1, 1, 1, 1, 1, 1]
+  - name: awkward_content_reduce_zeroparents_64
+    tests:
+    - error: true
+      inputs: {}
```

Comment:
I think, `length` in this case is `0` and the test should pass.

Reply:
The definition is,
```
def awkward_content_reduce_zeroparents_64(toparents, length):
    for i in range(length):
        toparents[i] = 0
```
So `length=0` will not change `toparents` but if `length=1` then `toparents=[0]` and the test will pass. Should I put the length as 0 or 1 or should I keep it as an error case?